documents/policies/internal/Contractor Compliance.md

```diff
@@ -0,0 +1,30 @@
+# WCA Contractor Compliance Policy
+
+### Version 1.0 {.version}
+
+## Purpose
+The purpose of this policy is to define how the WCA handles contractors who provide services for the organization in return for monetary benefit. 
+
+## Policy
+1. The Board in its absolute discretion has the ability to determine the need for the WCA to seek assistance from a contractor in delivering key services for the functioning of the organization
```

Comment:
Clunky wording. Note on Board having absolute discretion is supported by Bylaws and does not need to be duplicated here. Suggesting this simplification:

```suggestion
1. The Board decides if outside help from a contractor is needed to deliver services for the organization's operations.
```